languages/javascript/exercises/shared/.docs/debug.md

```diff
@@ -0,0 +1,7 @@
+# Debug
+
+When a test fails, a message is displayed describing what went wrong and for which input. You can also use the fact that any `console` output will be shown too. You can write to the console using:
+
+```javascript
+console.log('Debug message')
```

Comment:
OOI, is this implemented in the Test Runner?

Reply:
It is, but it isn't working quite right yet XD